faction/.gitignore

```diff
@@ -26,4 +26,4 @@
 *.iml
 *.ipr
 *.iws
-.idea/
\ No newline at end of file
+.idea/
```

Comment:
This was merely done so the github engine would recognize the change in the .gitignore file. I'd converted it to ANSI formatting to be compatable with the git engine.